query-node/schemas/contentNft.graphql

```diff
@@ -72,6 +72,11 @@ type OwnedNft
   "NFT transactional status"
   transactionalStatus: TransactionalStatus!
```

Comment:
I think if we're going this way, we should flatten all of the variants, not only the auction one. As it is, you can use both `transactionalStatus.auction` and `transactionalStatusAuction` for accessing the same data which may be confusing

Reply:
I think if we just keep it this way then it will be easier to roll back this change when we make the fix in Hydra and `transactionalStatus.auction` starts working, i.e we only need to remove the `transactionalStatusAuction` field and make a few changes in mappings. 

> we should flatten all of the variants 

Also, I don't see why this is necessary since we don't have any problem with other variants apart from `transactionalStatusAuction`. You are right that having both `transactionalStatus.auction` and `transactionalStatusAuction` for accessing the same data may be confusing but would it create technical problems/challenges in Atlas?